railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -14,6 +14,11 @@ Rails.application.configure do
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  # Aborts server start when missing the key to decrypt secrets in
+  # `config/secrets.yml.enc`. Assign the encryption key to
+  # `ENV["RAILS_MASTER_KEY"]`.
+  config.raise_on_missing_encryption_key = true
```

Comment:
`read_encrypted_secrets` ?

Seems needlessly confusing to silently load them in development just because the key is present